.github/workflows/wheel_linux_x86_64.yml

```diff
@@ -6,7 +6,7 @@ name: Wheel::Linux::x86_64
 # **Who does it impact**: Wheels to be uploaded to PyPI.
 
 env:
-  GCC_VERSION: 11
+  GCC_VERSION: 12
```

Comment:
@AmintorDusko consider using gcc-13 instead.

Reply:
Thanks, I will look into that.